### PR TITLE
Feature/13 list search results

### LINKED ISSUE #13: List of search results
As a user, I want to see a list of matching results after I clicked the search button, so that I can decide myself which list item is the one I was searching for.

### PR DESCRIPTION
**Pull request by maximilian-franz**
Closes #13

### CODE REVIEW

**Comment by Storyxx** [comment]
@chrisma Why exactly is `codeclimate` broken? Clicking on details doesn't seem to lead to any problems with our code, so what is the issue here?

**Comment by chrisma** [comment]
Hi @Storyxx, CodeFactor seems to work fine on this branch as far as I can tell.

File: app/controllers/search_results_controller.rb [hunk 1]
```diff
@@ -0,0 +1,49 @@
+class SearchResultsController < ApplicationController
+  before_action :set_search_result, only: %i[show edit update destroy]
+
+  # GET /search_results or /search_results.json
```
Comment (johannes-poetzsch) [inline app/controllers/search_results_controller.rb:4]: GET /search_results.json is not an available route so it should not be listed here.

File: app/controllers/search_results_controller.rb [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    # SearchResults are currently not stored a database so no need to edit one.
+  end
+
+  def update
+    # SearchResults are currently not stored a database so no need to update one.
+  end
+
+  def destroy
+    # All SearchResult objects will be destroyed after they are no longer used.
+  end
```
Comment (johannes-poetzsch) [inline app/controllers/search_results_controller.rb:36]: According to the [Ruby Style Guide](https://rubystyle.guide/) empty methods should be written like: `def destroy; end`.

**Review by johannes-poetzsch — APPROVED** [review]

File: app/controllers/search_results_controller.rb [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+  private
+
+  # Use callbacks to share common setup or constraints between actions.
+  def set_search_result
+    @search_result = SearchResult.find(params[:id])
+  end
+
+  # Only allow a list of trusted parameters through.
+  def search_result_params
+    params.fetch(:search_result, {})
```
Comment (johannes-poetzsch) [inline app/controllers/search_results_controller.rb:43]: Use `require` and `permit` to define which parameters should be accepted. More Information can be found [here](https://guides.rubyonrails.org/v6.0/getting_started.html#saving-data-in-the-controller).

**Review by johannes-poetzsch — CHANGES_REQUESTED** [review]

**Review by johannes-poetzsch — APPROVED** [review]

**Review by johannes-poetzsch — APPROVED** [review]

**Pull request merged**